src/Magnum/Platform/GlfwApplication.h

```diff
@@ -436,6 +436,31 @@ class GlfwApplication {
         /** @copydoc Sdl2Application::redraw() */
         void redraw() { _flags |= Flag::Redraw; }
 
+    public:
+        enum class Cursor : UnsignedInt {
```

Comment:
Document this enum as well? :) Same for SDL.

I'm usually adding a reference to the setter there, so people can jump from the enum directly to where it gets used.